test/functional/page_objects/common_page.js

```diff
@@ -37,6 +37,28 @@ export function CommonPageProvider({ getService, getPageObjects }) {
   const defaultFindTimeout = config.get('timeouts.find');
 
   class CommonPage {
+
+    static async navigateToUrlAndHandleAlert(url, shouldAcceptAlert) {
+      log.debug('Navigate to: ' + url);
```

Comment:
duplicate log message. The callers of this function already spit out a similar message. Maybe just remove log statement from this function?

Reply:
@thomasneirynck Using the generic text is ok.  However, the wording "Changes you made may not be saved" implies that even if the user previously hit the Save button, those changes might not be saved. If we can make a change, I'd suggest this text "Your unsaved changes might not be saved."

Reply:
Also the browser text is different for different browser's so the message is different in firefox. I am sure the internationalized browser text is also different so maybe we should not try to match it at all since it will be inconsistent in most cases